### PR TITLE
Add lang tags for some georef files

### PR DESCRIPTION
**Pull request by themerekat**
# Pull Request Checklist:
Adds some missing lang tags. Pretty sure geolocatetools isn't an active page, but I still went ahead and added the lang tags for the future.

# Pre-Approval

- [x] There is a description section in the pull request that details what the proposed changes do. It can be very brief if need be, but it ought to exist.
- [x] Features and backlog bugs should be merged into the `Development` branch, **NOT** `master`
- [x] All new text is preferably internationalized (i.e., no end-user-visible text is hard-coded on the PHP pages), and the [spreadsheet tracking internationalizations](https://docs.google.com/spreadsheets/d/133fps9w2pUCEjUA6IGCcQotk7dn9KvepMXJ2IWUZsE8/edit?usp=sharing) has been updated either with a new row or with checkmarks to existing rows.
- [x] There are no linter errors
- [x] [Symbiota coding standards](https://docs.google.com/document/d/1-FwCZP5Zu4f-bPwsKeVVsZErytALOJyA2szjbfSUjmc/edit?usp=sharing) have been followed

# Post-Approval

- [ ] It is the code author's responsibility to merge their own pull request after it has been approved
- [ ] If this PR represents a merge into the `Development` branch, remember to use the **squash & merge** option
- [ ] If this PR represents a merge into the `hotfix` branch, remember to use the **merge** option (i.e., no squash).
- [ ] If this PR represents a merge from the `Development` branch into the master branch, remember to use the **merge** option
- [ ] If this PR represents a merge from the `hotfix` branch into the `master` branch use the **squash & merge** option
  - [ ] a subsequent PR from `master` into `Development` should be made with the **merge** option (i.e., no squash).
  - [ ] **Immediately** delete the `hotfix` branch and create a new `hotfix` branch
  - [ ] increment the Symbiota version number in the symbase.php file and commit to the `hotfix` branch.
- [ ] If the dev team has agreed that this PR represents the last PR going into the `Development` branch before a tagged release (i.e., before an imminent merge into the master branch), make sure to notify the team and [lock the `Development` branch](https://github.com/BioKIC/Symbiota/settings/branches) to prevent accidental merges while QA takes place. Follow the release protocol [here](https://github.com/BioKIC/Symbiota/blob/master/docs/release-protocol.md).
- [ ] Don't forget to delete your feature branch upon merge. Ignore this step as required.

Thanks for contributing and keeping it clean!

### CODE REVIEW

File: content/lang/collections/georef/geolocate.es.php [hunk 1]
```diff
@@ -5,4 +5,7 @@
 ------------------
 */
 $LANG['GEO_LOCATE_TOOL'] = 'Herramienta GEOLocate';
+$LANG['IFRAME PERMISSION'] = 'la URL del iframe no tiene permiso para interactuar conmigo';
```
Comment (NikitaSalikov) [inline content/lang/collections/georef/geolocate.es.php]: Missing the underline "_" symbol

File: content/lang/collections/georef/geolocate.fr.php [hunk 2]
```diff
@@ -5,5 +5,7 @@
 ------------------
 */
 $LANG['GEO_LOCATE_TOOL'] = 'Outil GEOLocate';
+$LANG['IFRAME PERMISSION'] = "l'url iframe n'a pas la permission d'interagir avec moi";
```
Comment (NikitaSalikov) [inline content/lang/collections/georef/geolocate.fr.php]: Missing the underline "_" symbol

**Review by NikitaSalikov — CHANGES_REQUESTED** [review]
You missed the underscore for "IFRAME_PERMISSION" in es and fr lang files. Other than that - Looks good!

**Comment by themerekat** [comment]
Ah, pesky Google Translate removing my underscores!

**Review by NikitaSalikov — APPROVED** [review]

**Pull request merged**